### ISSUE TITLE
Catching multiple exceptions causes multiple false-positives

### ISSUE THREAD

**Issue by tucked** [PyCQA/pylint#2841]
This is essentially a reopen of #1756 and #2431 (with a false-positive E1101 in between).

### Steps to reproduce
``` python
"""example.py"""


def _foobar():
    exceptions = tuple([ValueError, TypeError])
    try:
        raise ValueError
    except exceptions as exc:
        print(exc.message)
        raise exc
```

### Current behavior
```
$ pylint example.py
************* Module example
example.py:8:11: E0712: Catching an exception which doesn't inherit from Exception: exceptions (catching-non-exception)
example.py:9:14: E1101: Instance of 'tuple' has no 'message' member (no-member)
example.py:10:8: E0710: Raising a new style class which doesn't inherit from BaseException (raising-non-exception)

----------------------------------------------------------------------
Your code has been rated at -11.43/10 (previous run: -11.43/10, +0.00)
```

### Expected behavior
No errors

### pylint --version output
```
pylint 2.3.1
astroid 2.2.5
Python 3.7.1 (default, Nov  6 2018, 20:20:54)
[GCC 4.8.5 20150623 (Red Hat 4.8.5-28)]
```

**Comment by PCManticore** [comment]
Hey @tucked Thanks for opening an issue. This should be fixed now in the astroid's `master`.
